ci/dra_upload.sh

```diff
@@ -70,11 +70,9 @@ gsutil cp gs://logstash-ci-artifacts/dra/${STACK_VERSION}/logstash-oss-${STACK_V
 gsutil cp gs://logstash-ci-artifacts/dra/${STACK_VERSION}/logstash-${STACK_VERSION}-arm64.deb build/
 gsutil cp gs://logstash-ci-artifacts/dra/${STACK_VERSION}/logstash-${STACK_VERSION}-aarch64.rpm build/
 gsutil cp gs://logstash-ci-artifacts/dra/${STACK_VERSION}/logstash-${STACK_VERSION}-linux-aarch64.tar.gz build/
-gsutil cp gs://logstash-ci-artifacts/dra/${STACK_VERSION}/logstash-${STACK_VERSION}-darwin-aarch64.tar.gz build/
```

Comment:
The scripts already know which version it is building, or uploading, and as such, I would much rather have a conditional here that checks if the current instance is 7.17 in order to run this command, instead of diverging branches from main and 7.17. 

This makes the maintenance significantly easier.

Reply:
Superseeded by #14677